entity-services/src/test/resources/model-units/instance-converter.xqy

```diff
@@ -235,37 +274,8 @@ declare function et-required:instance-to-envelope(
     $entity-instance as map:map
 ) as document-node()
 {
-    et-required:instance-to-xml-envelope($entity-instance)
+    et-required:instance-to-envelope($entity-instance, "xml")
```

Comment:
I'll push back on this -- in one release the comment will seem like cruft.  Also, instance-to-xml-envelope is still a perfectly valid function, even though we chose to implement the alternate calling mechanism.  I looked for ways to remove it, but it seemed like it was just putting complexity in for no good purpose.